src/components/StaffPayment/StaffPayment.vue

```diff
@@ -44,15 +44,14 @@
               @focus="paymentOption = StaffPaymentOptions.BCOL"
               @input="emitStaffPaymentData({ option: StaffPaymentOptions.BCOL, datNumber: $event })"
             />
-            <v-text-field
-              filled
-              id="folio-number-textfield"
-              label="Folio Number (Optional)"
-              :value="staffPaymentData.folioNumber"
-              :rules="folioNumberRules"
+            <FolioNumberInput
```

Comment:
Since this is a "significant" change to this shared component, do you think we should increase its version to 1.2.0 or 2.0.0 ?
(Is it fully compatible with the previous version?)

Reply:
I don't think so. It's totally compatible with the previous version.